incubator/parse/README.md

```diff
@@ -0,0 +1,112 @@
+| `parseServer.imagePullPolicy`      | Parse image pull policy                | `Always` if `image` tag is `latest`, else `IfNotPresent` |
+| `parseServer.port`                 | Parse server port                      | `1337`                                                   |
+| `parseServer.mountPath`            | Parse API mount path                   | `/parse`                                                 |
+| `parseServer.appId`                | Parse server App Id                    | `myAppID`                                                |
+| `parseServer.masterKey`            | Parse server Master Key                | `mymasterKey`                                            |
+| `parseServer.resources`            | CPU/Memory resource requests/limits    | Memory: `512Mi`, CPU: `300m`                             |
+| `parseDashboard.enabled`           | Enable parse dashboard                 | `true`                                                   |
+| `parseDashboard.image`             | Dashboard image                        | `bitnami/parse-dashboard:{VERSION}`                      |
+| `parseDashboard.imagePullPolicy`   | Dashboard image pull policy            | `Always` if `image` tag is `latest`, else `IfNotPresent` |
+| `parseDashboard.username`          | Dashboard username                     | `user`                                                   |
+| `parseDashboard.password`          | Dashboard user password                | `random 10 character alphanumeric string`                |
+| `parseDashboard.appName`           | Dashboard application name             | `MyDashboard`                                            |
+| `parseDashboard.resources`         | CPU/Memory resource requests/limits    | Memory: `512Mi`, CPU: `300m`                             |
+| `persistence.enabled`              | Enable Parse persistence using PVC     | `true`                                                   |
+| `persistence.storageClass`         | PVC Storage Class for Parse volume     | `generic`                                                |
+| `persistence.accessMode`           | PVC Access Mode for Parse volume       | `ReadWriteOnce`                                          |
+| `persistence.size`                 | PVC Storage Request for Parse volume   | `8Gi`                                                    |
+| `mongodb.persistence.enabled`      | Enable MongoDB persistence using PVC   | `true`                                                   |
+| `mongodb.persistence.storageClass` | PVC Storage Class for MongoDB volume   | `generic`                                                |
+| `mongodb.persistence.accessMode`   | PVC Access Mode for MongoDB volume     | `ReadWriteOnce`                                          |
+| `mongodb.persistence.size`         | PVC Storage Request for MongoDB volume | `1Gi`                                                    |
```

Comment:
Value must be `8Gi` as stated in values.yaml
https://github.com/bitnami/charts/pull/208/files#diff-7bc76d1c66d6209e83a32ab5b02a0fd2R111